vars/provisionNodes.groovy

```diff
@@ -163,239 +126,180 @@ def call(Map config = [:]) {
+                            rc=\\${PIPESTATUS[0]}
+                            for file in /etc/yum.repos.d/*.repo; do
+                              echo \\"---- \\$file ----\\"
+                              cat \\$file
+                            done
+                            exit \\$rc
+                        fi
+                        if [ ! -e /usr/bin/pip3 ] &&
+                           [ -e /usr/bin/pip3.6 ]; then
+                            ln -s pip3.6 /usr/bin/pip3
+                        fi
+                        if [ ! -e /usr/bin/python3 ] &&
+                           [ -e /usr/bin/python3.6 ]; then
+                            ln -s python3.6 /usr/bin/python3
+                        fi'''
+  } else if (distro_type == "suse") {
+    if (repository_g != '') {
+      provision_script += '\nzypper --non-interactive ar -f ' +
+                           repository_g + ' daos-stack-group-repo'
+      provision_script += '\nzypper --non-interactive mr ' +
+                          '--gpgcheck-allow-unsigned-repo ' +
+                          'daos-stack-group-repo'
+    }
+    if (repository_l != '') {
+      provision_script += '\nzypper --non-interactive ar' +
+                          ' --gpgcheck-allow-unsigned -f ' +
+                          repository_l + ' daos-stack-local-repo'
+      provision_script += '\nzypper --non-interactive mr' +
                             ' --no-gpgcheck daos-stack-local-repo'
-      }
-      if (inst_repos) {
-        provision_script +=   '\n' + iterate_repos +
-                            '''\n    zypper --non-interactive ar --gpgcheck-allow-unsigned -f ''' + env.JENKINS_URL + '''job/daos-stack/job/\\\${repo}/job/\\\${branch//\\//%252F}/\\\${build_number}/artifact/artifacts/leap15/ \\\$repo
-                                 done'''
-      }
-      provision_script += '\nzypper --non-interactive' +
-                          ' --gpg-auto-import-keys --no-gpg-checks ref'
-      provision_script += '\nzypper --non-interactive in ed nfs-client '
-      provision_script += 'ipmctl sudo '
-      if (inst_rpms) {
-        provision_script += inst_rpms
-      }
-    } else {
-        error("Don't know how to handle repos for distro: \"" + distro + "\"")
     }
-    provision_script += '''\nsync; sync"
-                           exit ${PIPESTATUS[0]}'''
-    def rc = 0
-    rc = sh(script: provision_script,
-            label: "Post provision configuration",
-            returnStatus: true)
-    if (rc != 0) {
-      stepResult name: env.STAGE_NAME, context: "test", result: "FAILURE"
-      error "One or more nodes failed post-provision configuration!"
+    if (inst_repos) {
+      provision_script +=   '\n' + iterate_repos +
+                          '''\n    zypper --non-interactive ar --gpgcheck-allow-unsigned -f ''' + env.JENKINS_URL + '''job/daos-stack/job/\\\${repo}/job/\\\${branch//\\//%252}/\\\${build_number}/artifact/artifacts/leap15/ \\\$repo
```

Comment:
Looks like the _F_ got dropped here.  This line should just not change in this patch.  This reverts this change:
```suggestion
                          '''\n    zypper --non-interactive ar --gpgcheck-allow-unsigned -f ''' + env.JENKINS_URL + '''job/daos-stack/job/\\\${repo}/job/\\\${branch//\\//%252F}/\\\${build_number}/artifact/artifacts/leap15/ \\\$repo
```